labs/aifaq.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: AI FAQ for Hyperledger Ecosystem
+[//]:# (parent: Labs)
+---
+
+# aifaq
+
+AI FAQ for Hyperledger Ecosystem
+
+# Short Description
+
+AI FAQ provides a ChatBot that replies to the questions on Hyperledger Ecosystem: that is a Generative Pre-Trained Model based on Transformer technology. The system has a modern QA architecture based on Retriever-Reader pipeline. The Retriever component has the purpose of select the most relevant documents, based on similarity of the question. While the Reader component returns the answer to the user, with a snippet of text extracted from the documentation. The final version will use the most sophisticated technologies (RAG, QLoRA, DPO, ...).
+
+# Scope of Lab
+
+The scope of the lab is to support the Hyperledger users (users, developer, etc.) to their work, avoiding to wade through oceans of documents to find information they are looking for. Large Language Models have yielded remarkable results, either pay and open source tools. Today we can implement a conversational AI tool which replies to questions related to specific context.
+
+# Initial Committers
+
+https://github.com/gcapuzzi
+
+# Sponsor
+
+- Tracy A. Kuhrt - https://github.com/tkuhrt
+
+# Pre-existing repository
+
+- https://github.com/gcapuzzi/hyperledger_qa_poc
```

Comment:
In order to bring an existing source base to the Hyperledger Foundation, the repository must have [DCO](https://developercertificate.org/) signoff on all commits. Since your existing repository does not have DCO signoff on all commits, you will need to do one of two things:

1. bring your code by squashing all of your commits into a single first commit made against a new lab repo with your DCO sign-off.

1.  amend the commit history to include DCO sign-off for each of the commits. The Hyperledger Indy community has [documented steps to fix DCO on previous commits](https://github.com/hyperledger/indy-sdk/blob/main/docs/contributors/signing-commits.md#how-to-sign-previous-commits). Also, the [Fix DCO Guide from src-d](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) contains some different steps you can take.

Please let us know which option you would like to take for this repository.

Reply:
I will check thank you

Reply:
My existing source base has three commits: I'm checking the documentation on how "to fix DCO on previous commits"

Reply:
Great! I am approving this proposal. Let's make sure that we have this resolved prior to moving the code over to Hyperledger.

Reply:
Please check now that repo https://github.com/gcapuzzi/hyperledger_qa_poc it should have all commits ok. Thanks a lot

Reply:
Done